cwltool/schemas/v1.0/Process.yml

```diff
@@ -562,7 +562,9 @@ $graph:
 
     - name: default
       type: Any?
-      jsonldPredicate: "cwl:default"
+      jsonldPredicate:
```

Comment:
Yes, please send a new pull request to the other repo, with these changes applied to the `v1.0` and `v1.1.0-dev1` folders.